Gemfile

```diff
@@ -36,6 +36,7 @@ group :development, :test do
 end
 
 group :test do
+  gem 'climate_control', '~>0.1'
```

Comment:
one very small change here just add a space between the `~>` and the version number